src/cljs/org/broadinstitute/firecloud_ui/common.cljs

```diff
@@ -1,4 +1,7 @@
-(ns org.broadinstitute.firecloud-ui.common)
+(ns org.broadinstitute.firecloud-ui.common
+  (:require
+   [org.broadinstitute.firecloud-ui.utils :as utils]
```

Comment:
No. It's just annoying to add this line when you want to use the debugging tools (which is exactly what happened and why I left it in).